public/landing-page.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins&display=swap">
+        <link rel="stylesheet" href="landing-page.css">
+    </head>
+    <body class="body-gradient">
+        <nav class="landing-header drop-shadow">
+            <div class="logo-container"></div>
+            <div class="nav-container">
+                <div class="nav-item-link">
+                    <div class="nav-link">
+                        About
+                    </div>
+                </div>
+                <div class="nav-item-link">
+                    <div class="nav-link">
+                        Log In
+                    </div>
+                </div>
+                <div class="nav-item-button">
+                    <div class="drop-shadow primary-button-sleek font-white" onclick="alert('Join Party!')">Join Party</div>
+                </div>
+                <div class="nav-item-button">
+                    <div class="drop-shadow primary-button-sleek font-white" onclick="alert('Watch Party!')">Host Party</div>
+                </div>
+                <div class="nav-item-profile">
+                    <img class="profile-avatar" src="../src/assets/profile_pic.svg" alt="Avatar" >
+                </div>
+            </div>
+        </nav>
+        <main class="landing-body">
+            <section class="section section-one">
+                <img class=" landing-sec-1-img" src = "../src/assets/sample_party_screen.svg" alt="sample_party_screen"/>
+                <div class="landing-sec-1-heading font-white">
+                    Welcome to <br>the NextGen theater
+                </div>
+                <div class="landing-sec-1-heading-divider"></div>
+                <div class="landing-sec-1-sub-heading font-white">
+                    Host video parties across the major streaming services, and <br>
```

Comment:
Let's constrain the container width if want to divide the text into multiple lines, or you can also use `span` tags.